deploy/amp/database_initialise.php

```diff
@@ -8,9 +8,9 @@
 // james: uncomment the following lines to also initialise the
 //        primary 'buttonmen' database
 //
-//        recreate_database('buttonmen');
-//        $conn = db_connect('buttonmen');
-//        run_all_sql_files($conn);
+        recreate_database('buttonmen');
+        $conn = db_connect('buttonmen');
+        run_all_sql_files($conn);
```

Comment:
Note that you've committed this change here - i don't care, nothing live uses this file, just mentioning it in case you do care and didn't mean to do that, in which case revert it the next time you push a change.